Makefile

```diff
@@ -62,8 +62,16 @@ nb: clean
 	python3 -m venv .venv
 	. .venv/bin/activate && python -m pip install .[all_extras,binder] && ./build_tools/run_examples.sh
 
-dockerbuild:
-	DOCKER_BUILDKIT=1 docker build -t sktime -f build_tools/docker/Dockerfile .
+dockertest:
+
+ifeq ($(pyversion),3.7)
```

Comment:
How about replacing the if block with a variable passed to the command, sth like 

```
docker build -t sktime -f build_tools/docker/$(PYTHON_VERSION).dockerfile .
```

where `PYTHON_VERSION` would be `py39`